### DISCUSSION TITLE
Writing tests in TS

### DISCUSSION THREAD

**Discussion by colbyfayock** [cloudinary-community/cloudinary-util#141] [Ideas]
Question from https://github.com/colbyfayock/cloudinary-util/pull/139#discussion_r1484063947

> suggestion: this is a rather general suggestion but have you maybe thought about writing spec files in .ts instead of .js?
> We are using it in both my work and personal/community projects and it has proven to be quite useful :)

Is it as simple as renaming tests to .ts? What are the implications and workflow differences?

**Comment by Baroshem** [comment]
You are using Vitest which supports both .js and .ts files. There might be a simple configuration needed if just renaming wont work but I see a big value in being able to have type support in the test files as well :)
